x/evm/keeper/keeper.go

```diff
@@ -189,11 +191,9 @@ func (k *Keeper) GetVMBlockContext(ctx sdk.Context, gp core.GasPool) (*vm.BlockC
 	if err != nil {
 		return nil, err
 	}
-	r, err := ctx.BlockHeader().Time.MarshalBinary()
-	if err != nil {
-		return nil, err
-	}
-	rh := common.BytesToHash(r)
+
+	timestamp := ctx.BlockHeader().Time.Unix()
```

Comment:
nit: can we take the hash of the exact block timestamp of this block?

Reply:
Updated